### PR TITLE
 LESQ-62 Feat/examboard unit listing

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

add exam board to unit listing title if exist

## Issue(s)

Fixes #

## How to test

can only be testing when corresponding db-tools ticket is merged

1. Go to https://deploy-preview-1737--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/history-secondary-ks4-aqa/units
3. You should see examboard title in programme title

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by jack-skerrett-oak — APPROVED** [review]
KS4 history/english both display exam boards when you navigate to the unit listing page for programmes with exam boards!

File: src/node-lib/curriculum-api-2023/queries/unitListing/unitListing.gql [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-    subjectTitle: subject_title
-    tierSlug: tier_slug
-    totalUnitCount: unit_count
+    programmeSlug
+    keyStageSlug
+    keyStageTitle
+    subjectSlug
+    subjectTitle
+    tierSlug
+    totalUnitCount:unitCount
```
Comment (JBR90) [inline src/node-lib/curriculum-api-2023/queries/unitListing/unitListing.gql:13]: Should we be transforming the name here to match 2020 content?

**Review by JBR90 — APPROVED** [review]
Looks good, one question.

**Pull request merged**